spec/dummy/db/migrate/20190829220515_add_message_type_to_system_messages.rb

```diff
@@ -0,0 +1,7 @@
+class AddMessageTypeToSystemMessages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :system_messages, :message_type, :integer, null: false
```

Comment:
existing system messages in production will cause this to crash since they have nils

Reply:
nevermind, this is RC